### PR TITLE
Update node conformance test document.

### PR DESCRIPTION
**Pull request by Random-Liu**
Update node conformance test document.

@dchen1107 
/cc @kubernetes/sig-node

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/1784)

### CODE REVIEW

File: docs/admin/node-conformance.md [hunk 1]
```diff
@@ -9,54 +9,52 @@ assignees:
...
-conformance test. They'll be improved in future version.
+conformance test.
 
 * Node conformance test only supports Docker as the container runtime.
-* Node conformance test doesn't validate network related system configurations
-  and functionalities.
 
 ## Prerequisite
 
 Node conformance test is used to test whether a node is ready to join a
```
Comment (devin-donnelly) [inline docs/admin/node-conformance.md]: "To run node conformance test, a node must satisfy the same prerequisites as a standard Kubernetes node. At a minimum, the node should have the following processes installed:"

File: docs/admin/node-conformance.md [hunk 2]
```diff
@@ -9,54 +9,52 @@ assignees:
...
+node. At least, the node should have installed:
 
 * Container Runtime (Docker)
 * Kubelet
 
-Node conformance test validates kernel configurations. If the kenrel module
-`configs` is built as module in your environment, it must be loaded before the
-test. (See [Caveats #3](#caveats) for more information)
-
 ## Usage
```
Comment (devin-donnelly) [inline docs/admin/node-conformance.md]: Try not to nest headers underneath one another. That's usually indicative of structural problems in the document.

In this case, "## Usage" is unnecessary. Remove, and move all subsequent headers up one level.

File: docs/admin/node-conformance.md [hunk 3]
```diff
@@ -9,54 +9,52 @@ assignees:
...
 
 ### Run Node Conformance Test
```
Comment (devin-donnelly) [inline docs/admin/node-conformance.md]: Add an introductory sentence:

"To run a node conformance test, perform the following steps:"

File: docs/admin/node-conformance.md [hunk 4]
```diff
@@ -9,54 +9,52 @@ assignees:
...
 
 * **Step 1:** Point your Kubelet to localhost `--api-servers="http://localhost:8080"`,
```
Comment (devin-donnelly) [inline docs/admin/node-conformance.md]: You can remove "**Step 1:**" and the bullet list, and use a markdown list instead. You can still nest bullets inside of it for the Kubelet flags.

1. Point your Kubelet to localhost, etc.
  * `--pod-cidr`: If you are using `kubenet`...
2. Run the node conformance test with the following command:

File: docs/admin/node-conformance.md [hunk 5]
```diff
@@ -9,54 +9,52 @@ assignees:
...
 * Container Runtime (Docker)
 * Kubelet
 
-Node conformance test validates kernel configurations. If the kenrel module
-`configs` is built as module in your environment, it must be loaded before the
-test. (See [Caveats #3](#caveats) for more information)
-
 ## Usage
 
 ### Run Node Conformance Test
```
Comment (devin-donnelly) [inline docs/admin/node-conformance.md]: "Run" -> "Running"

File: docs/admin/node-conformance.md [hunk 6]
```diff
@@ -9,54 +9,52 @@ assignees:
...
-  -v $CONFIG_DIR:/etc/manifest -v $LOG_DIR:/var/result \
-  gcr.io/google_containers/node-test-amd64:v0.1
+  -v /:/rootfs -v $CONFIG_DIR:$CONFIG_DIR -v $LOG_DIR:/var/result \
+  gcr.io/google_containers/node-test:0.2
 ```
 
 ### Run Node Conformance Test for Other Architectures
 
-We also build node conformance test docker images for other architectures:
+Find node conformance test docker images for other architectures:
```
Comment (devin-donnelly) [inline docs/admin/node-conformance.md]: "Kubernetes also provides node conformance test docker images for other architectures:"

File: docs/admin/node-conformance.md [hunk 7]
```diff
@@ -66,49 +64,37 @@ We also build node conformance test docker images for other architectures:
...
+
 ### Caveats
 
-* The test will leave some docker images on the node, including the node
-  conformance test image and images of containers used in the functionality
+* The test leaves some docker images on the node, including the node conformance
+  test image and images of containers used in the functionality
   test.
-* The test will leave dead containers on the node, these containers are created
+* The test leaves dead containers on the node, these containers are created
```
Comment (devin-donnelly) [inline docs/admin/node-conformance.md]: Replace comma with period. Split this into two sentences.

File: docs/admin/node-conformance.md [hunk 8]
```diff
@@ -9,54 +9,52 @@ assignees:
 
 ## Node Conformance Test
 
-*Node conformance test* is a test framework validating whether a node meets the
-minimum requirement of Kubernetes with a set of system verification and
-functionality test. A node which passes the tests is qualified to join a
-Kubernetes cluster.
+*Node conformance test* is a containerized test framework validating whether a
```
Comment (Random-Liu) [inline docs/admin/node-conformance.md]: Will do.

File: docs/admin/node-conformance.md [hunk 9]
```diff
@@ -9,54 +9,52 @@ assignees:
...
+node meets the minimum requirement of Kubernetes with a set of system
+verification and functionality test. A node which passes the test is qualified
+to join a Kubernetes cluster.
 
 ## Limitations
 
 There are following limitations in the current implementation of node
```
Comment (Random-Liu) [inline docs/admin/node-conformance.md]: Good suggestion! Will do.

File: docs/admin/node-conformance.md [hunk 10]
```diff
@@ -9,54 +9,52 @@ assignees:
...
-conformance test. They'll be improved in future version.
+conformance test.
 
 * Node conformance test only supports Docker as the container runtime.
-* Node conformance test doesn't validate network related system configurations
-  and functionalities.
 
 ## Prerequisite
```
Comment (Random-Liu) [inline docs/admin/node-conformance.md]: Will do.

File: docs/admin/node-conformance.md (hunk 5 above)
Comment (Random-Liu) [inline docs/admin/node-conformance.md]: Will do.

File: docs/admin/node-conformance.md (hunk 1 above)
Comment (Random-Liu) [inline docs/admin/node-conformance.md]: Will do.

File: docs/admin/node-conformance.md (hunk 2 above)
Comment (Random-Liu) [inline docs/admin/node-conformance.md]: Will do.

File: docs/admin/node-conformance.md (hunk 4 above)
Comment (Random-Liu) [inline docs/admin/node-conformance.md]: Will do.

File: docs/admin/node-conformance.md (hunk 6 above)
Comment (Random-Liu) [inline docs/admin/node-conformance.md]: Will do.

File: docs/admin/node-conformance.md (hunk 7 above)
Comment (Random-Liu) [inline docs/admin/node-conformance.md]: Will do.

File: docs/admin/node-conformance.md (hunk 8 above)
Comment (Random-Liu) [inline docs/admin/node-conformance.md]: Done.

File: docs/admin/node-conformance.md (hunk 9 above)
Comment (Random-Liu) [inline docs/admin/node-conformance.md]: Done.

File: docs/admin/node-conformance.md (hunk 10 above)
Comment (Random-Liu) [inline docs/admin/node-conformance.md]: Done.

File: docs/admin/node-conformance.md (hunk 1 above)
Comment (Random-Liu) [inline docs/admin/node-conformance.md]: Done.

File: docs/admin/node-conformance.md (hunk 2 above)
Comment (Random-Liu) [inline docs/admin/node-conformance.md]: Done.

File: docs/admin/node-conformance.md (hunk 5 above)
Comment (Random-Liu) [inline docs/admin/node-conformance.md]: Done..

File: docs/admin/node-conformance.md (hunk 3 above)
Comment (Random-Liu) [inline docs/admin/node-conformance.md]: Done.

File: docs/admin/node-conformance.md (hunk 4 above)
Comment (Random-Liu) [inline docs/admin/node-conformance.md]: Done.

File: docs/admin/node-conformance.md (hunk 6 above)
Comment (Random-Liu) [inline docs/admin/node-conformance.md]: Done.

File: docs/admin/node-conformance.md (hunk 7 above)
Comment (Random-Liu) [inline docs/admin/node-conformance.md]: Done.

**Comment by Random-Liu** [comment]
@devin-donnelly Addressed comments. Thanks a lot for your carefully review.

**Comment by devin-donnelly** [comment]
Awesome, thanks. Who's a good tech reviewer for this pull request?

**Comment by dchen1107** [comment]
LGTM

**Pull request merged**